### DISCUSSION TITLE
back to its default view angle

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#1981] [Q&A] [answered]
Hey,
is there a way to set the view angle of the model to its starting point, after the user stops interacting with it?

**Comment by elalish** [comment, marked as answer]
Sure, just set `mv.cameraOrbit = "auto auto auto"` or whatever other value you'd like. I guess put that in a mouseup/keyup handler, maybe with a delay?

**Reply by tom-deadline** [reply]
@elalish thanks for the answer!
`mv.cameraOrbit = "auto auto auto"` works as I need if I'm not using `auto-rotate`. With `auto-rotate` it stays at its current position.

**Reply by elalish** [reply]
Then you'll want `mv.resetTurntableRotation()`: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-methods-resetTurntableRotation
